src/kuberesources.ts

```diff
@@ -19,6 +19,7 @@ export const allKinds = {
     job: new ResourceKind("Job", "Jobs", "Job", "job"),
     cronjob: new ResourceKind("CronJob", "CronJobs", "CronJob", "cronjob"),
     pod: new ResourceKind("Pod", "Pods", "Pod", "pod"),
+    crd: new ResourceKind("CRD", "CRDs", "CRD", "crd"),
```

Comment:
To clarify: the first two arguments are about how to display the resource type, and can be a matter of taste.  The third **MUST** correspond to the `kind:` value in the k8s manifest YAML.  The fourth must be something you can pass to `kubectl get ...` to list instances.  It would not be a bad plan if I documented this better.

Reply:
fixing, thanks!